Docker/build/Elasticsearch/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG ELASTICSEARCH_VERSION
+FROM elasticsearch:${ELASTICSEARCH_VERSION}
+RUN ./bin/elasticsearch-plugin install org.wikimedia.search:extra:6.5.4
+RUN ./bin/elasticsearch-plugin install org.wikimedia.search.highlighter:experimental-highlighter-elasticsearch-plugin:6.5.4
```

Comment:
Should this also use ${ELASTICSEARCH_VERSION}?

Reply:
done!